.gitmodules

```diff
@@ -330,3 +330,6 @@
 [submodule "contrib/crc32-vpmsum"]
 	path = contrib/crc32-vpmsum
 	url = https://github.com/antonblanchard/crc32-vpmsum.git
+[submodule "contrib/idxd-config"]
```

Comment:
Could we move l. 333ff right after submodule "qpl" (l. 257?)

Also, please omit the ".git" suffix (l. 335)